### DISCUSSION TITLE
GUI vs. commands

### DISCUSSION THREAD

**Discussion by MichalisMihalitsis** [talmolab/sleap#1237] [Help!] [answered]
Hello everyone,
I am a new user of SLEAP. I can open SLEAP through the Anaconda Command prompt (sleap-label), and then get a 'Happy SLEAPing' message in the command prompt. But then I can't write anything else in the prompt.
I can see that the SLEAP website has many different functions in the 'Notebooks' section, so my question is, where do you write these if you can't write anything in conda after opening the SLEAP GUI?
Looking forward to hearing from you, and thanks!
Mike

**Comment by roomrys** [comment, marked as answer]
Hi @MichalisMihalitsis,

The `sleap-label` command will open the SLEAP GUI for labeling purposes (and take up that terminal - you will not be able to run any CLI through this terminal while the GUI is open). If you would like to use the SLEAP API, you will need to open another terminal, `conda activate sleap`, and then you can either open and run a provided notebook, create your own notebook/python file, or run some commands through `ipython`.

Let us know if that helps! (Colab is also a great option for running notebooks if that suits you.)

Thanks,
Liezl
